### PR TITLE
Merging BB changes into CMoore branch

### PR DESCRIPTION
**Pull request by BobBooth**
This should add the changes to Highet Points, Lowest Points, and Local
Peaks into Military Tools.pyt and VisTool.py. Also includes
InputArea_FeatureSetGDB.lyr output with rep paths using GP tool.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.

### CODE REVIEW

File: toolboxes/scripts/VisTools.py [hunk 1]
```diff
@@ -878,3 +878,227 @@ def execute(self, parameters, messages):
...
+
+    def getParameterInfo(self):
+        # Input_Area
+        param_1 = arcpy.Parameter()
+        param_1.name = 'Input_Area'
+        param_1.displayName = 'Input Area'
+        param_1.parameterType = 'Required'
+        param_1.direction = 'Input'
+        param_1.datatype = 'Feature Set'
+        #param_1.value = 'in_memory\\{35E2B4F4-CF01-4F72-A588-68AAE10EAF69}'
```
Comment (csmoore) [inline toolboxes/scripts/VisTools.py]: May want to remove this comment line

File: toolboxes/scripts/VisTools.py [hunk 2]
```diff
@@ -878,3 +878,227 @@ def execute(self, parameters, messages):
...
+        param_2.datatype = 'Raster Layer'
+
+        # Output_Lowest_Point_Features
+        param_3 = arcpy.Parameter()
+        param_3.name = 'Output_Lowest_Point_Features'
+        param_3.displayName = 'Output Lowest Point Features'
+        param_3.parameterType = 'Required'
+        param_3.direction = 'Output'
+        param_3.datatype = 'Feature Class'
+        param_3.value = '%scratchGDB%/lowestPoints'
```
Comment (csmoore) [inline toolboxes/scripts/VisTools.py]: Add symbology

File: toolboxes/scripts/VisTools.py [hunk 3]
```diff
@@ -878,3 +878,227 @@ def execute(self, parameters, messages):
...
+
+        # Output_Highest_Point_Features
+        param_3 = arcpy.Parameter()
+        param_3.name = 'Output_Highest_Point_Features'
+        param_3.displayName = 'Output Highest Point Features'
+        param_3.parameterType = 'Required'
+        param_3.direction = 'Output'
+        param_3.datatype = 'Feature Class'
+        param_3.value = '%scratchGDB%/highestPoints'
+
```
Comment (csmoore) [inline toolboxes/scripts/VisTools.py]: Add symbology

**Comment by BobBooth** [comment]
@csmoore - note GDB has updated schema to support the output lyr files. VisUtils, output LYR files with GDB suffix and relative paths, and GDB included in the latest commit.

**Pull request merged**